grails-app/assets/javascripts/utilsJavaScript/index.js

```diff
@@ -0,0 +1,79 @@
+function searchZipCode(value) {
+    let zipCode = value.replace(/\D/g, '')
+
+    if(isZipCodeEmpty(zipCode)) return
```

Comment:
```suggestion
    if (isZipCodeEmpty(zipCode)) return;
```
Padroniza os ifs em relação ao espaçamento